security/PracticalSecurity.md

```diff
@@ -0,0 +1,3 @@
+## Practical Security
+
+Today I changed some of my acocunts to use 2 factor authentication. I had only previously done this with one account and it is something I have been meaning to do for some time now but always skip the option when logging in. It makes sense to do this becasue my phone is usually in the same room as me and it is not a big inconvience to have to enter a code sent to my phone. I also changed some of my passwords to passphrases as I realize that most of my old passwords were easily crackable. While doing this I came to the relaization that a passphrase is actually easier for me to remember, and harder for someone to guess or crack than almost any passwords I have had in the past. I also wrote my passphrases down and stored them in secure place which will come in handy if I do forget one on them- or which one belongs to which account.
```

Comment:
Great start! I always recommend using a password manager, as they'll often take a lot of "grunt work" out of personal security. They help to provide secure password and have features that can automatically change your password on sensitive accounts and really simplify your login flow (often logging in automatically if your password manager is open)